drivers/gpio/Kconfig.da1469x

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2022 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_DA1469X
+	bool "DA1469x GPIO driver"
+	default y
```

Comment:
make this based on default `$(dt_compat_enabled,$(DT_COMPAT_RENESAS_DA1469X_GPIO))`